website/scripts/build-static-content.js

```diff
@@ -336,6 +337,49 @@ module.exports = {
                 }
               }
 
+              // Checking the metadata on /articles pages, and adding the category to the each article's URL.
+              if(sectionRepoPath === 'articles/') {
+                if(!embeddedMetadata.authorsGitHubUserName) {
```

Comment:
I think this would feel more predictable and less typo-prone if it was named `authorGitHubUsername` (no "s", lowercase "n" in username)